src/Restaurant.js

```diff
@@ -1,17 +1,14 @@
-import React from "react";
-
-const Restaurant = () => {
-  const pizzas = 0;
-  return (
-    <div>
-      <h3>Restaurant Orders</h3>
-      <ul>
-        <li>
-          Pizzas: {pizzas} <button className="btn btn-primary">Add</button>
-        </li>
-      </ul>
-    </div>
-  );
-};
-
-export default Restaurant;
+import React, { Component } from 'react'
+import Orders from './Orders'
+class Restaurant extends Component {
+  render () {
+    return (
+      <div>
+        <h3>Restaurant Orders</h3>
+        <Orders orderType='Pizza' />
+        <Orders orderType='Salad' />
```

Comment:
Can you see how making our components reusable (but configurable via props) is helpful?